Source/Networking/NetworkCore/NetPeer.cpp

```diff
@@ -2292,6 +2295,108 @@ void NetPeer::HandleReceivedUserAddResponse(NetPeerId theirNetPeerId,
   owner->DispatchEvent(Events::NetPeerReceivedUserAddResponse, &event);
 }
 
+NetUser* NetPeer::ServerForceAddUser(NetPeerId theirNetPeerId, EventBundle* addRequestBundle)
+{
+    Assert(IsOpen());
+    Assert(IsServerOrOffline());
+
+    // Get owner as game session
+    GameSession* owner = static_cast<GameSession*>(GetOwner());
+
+    // Create event
+    NetPeerReceivedUserAddRequest event(owner);
+    event.mTheirNetPeerId = theirNetPeerId;
+    event.mTheirRequestBundle = *addRequestBundle;
+
+    // Set active net user add request (used temporarily during net user creation)
+    mActiveUserAddRequest = &event;
+
+    // Dispatch event
+    owner->DispatchEvent(Events::NetPeerReceivedUserAddRequest, &event);
+
+    // Clear active net user add request
+    mActiveUserAddRequest = nullptr;
+
+    // User request accepted?
+    if (event.mReturnOurAddResponse)
+    {
+        // Get their net user object as provided by the event handler
+        Cog* cog = event.mReturnTheirNetUser;
+        if (!cog) // Unable?
```

Comment:
in general these kind of comments are not needed :)

Reply:
They're copied from another function in NetPeer, so I was just mirroring style really